package.json

```diff
@@ -32,11 +32,11 @@
     "dbus": "^1.0.7",
     "mdns-resolver": "^1.0.0",
     "semver": "^7.3.2",
-    "sqlite3": "^4.1.1",
+    "sqlite3": "^5.0.11",
```

Comment:
This is required for node >=14 but looks to cause failures on balenaCI related to it running an older docker version but looks to be fine with the `balena push` - I expect the best solution will likely be to migrate the supervisor to flowzone